Lib/test/test_subprocess.py

```diff
@@ -59,10 +59,14 @@ def setUp(self):
         support.reap_children()
 
     def tearDown(self):
-        for inst in subprocess._active:
-            inst.wait()
-        subprocess._cleanup()
-        self.assertFalse(subprocess._active, "subprocess._active not empty")
+        if not mswindows:
+            # subprocess._active is not used on Windows and is set to None.
+            for inst in subprocess._active:
+                inst.wait()
+            subprocess._cleanup()
+            self.assertFalse(
+                subprocess._active, "subprocess._active not empty"
+            )
         self.doCleanups()
         support.reap_children()
```

Comment:
Calling `support.read_children` is pointless but harmless in Windows. It has nothing to do:

    # Need os.waitpid(-1, os.WNOHANG): Windows is not supported
    if not (hasattr(os, 'waitpid') and hasattr(os, 'WNOHANG')):
        return

Reply:
Sorry, I don't quite understand why this is harmless. Could you please elaborate?

Reply:
Ah, sorry, misread harm*less* as harm*ful*.

Reply:
Victor had mentioned that calling `support.reap_children` was useful in Windows, but the snippet of code above shows that it doesn't do anything in Windows.